From c3478f8534b1fc3f6f19b8b912fe4509bd770449 Mon Sep 17 00:00:00 2001
From: Jiping Ma <jiping.ma2@windriver.com>
Date: Tue, 11 Jun 2024 06:49:13 +0000
Subject: [PATCH 3/3] i40e: pass linux common header

For debian, there are two header pkgs, e.g. linux-headers-6.6.0-1-amd64
and linux-headers-6.6.0-1-common. So pass linux-headers-6.6.0-1-common
to meet the requirement.

Signed-off-by: Jiping Ma <jiping.ma2@windriver.com>
[jpm: refresh the patch for v2.28.9]
Signed-off-by: Jiping Ma <jiping.ma2@windriver.com>
---
 src/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common.mk b/src/common.mk
index 8a4ae87..e9399b4 100644
--- a/src/common.mk
+++ b/src/common.mk
@@ -297,7 +297,7 @@ minimum_kver_check = $(eval $(call _minimum_kver_check,${1},${2},${3}))
 # call script that populates defines automatically
 $(if $(shell \
     $(if $(findstring 1,${V}),,QUIET_COMPAT=1) \
-    KSRC=${KSRC} OUT=${src}/kcompat_generated_defs.h CONFIG_FILE=${CONFIG_FILE} \
+    KSRC=${KSRC_COMMON} OUT=${src}/kcompat_generated_defs.h CONFIG_FILE=${CONFIG_FILE} \
     bash ${src}/kcompat-generator.sh && echo ok), , $(error kcompat-generator.sh failed))
 
 ################
-- 
2.42.0

